telegram_bot/database/create_connection.py

```diff
@@ -0,0 +1,22 @@
+from pydantic_settings import BaseSettings, SettingsConfigDict
```

Comment:
I think database module should be placed in project root directory, due to scrapper and bot use it both

Reply:
> I think database module should be placed in project root directory, due to scrapper and bot use it both

yea. Most likely, it's a bug of mine. I uploaded an old version of the structure to @nkorob